test/JDBC/expected/remote-proc-exec-vu-verify.out

```diff
@@ -0,0 +1,1211 @@
+GO
+~~START~~
+int#!#bigint#!#smallint#!#tinyint
+2147483647#!#9223372036854775807#!#32767#!#50
+~~END~~
+
+
+-- Test 6: FLOAT and REAL types
+-- Expected: Should preserve floating point values
+EXEC bbf_rpe_server.master.dbo.sp_TestFloat @p_float=3.14159265359, @p_real=2.71828;
+GO
+~~START~~
+float#!#real
+3.14159265359#!#2.71828
+~~END~~
+
+
+-- Test 7: FLOAT and REAL with variables
+-- Expected: Should handle typed FLOAT/REAL variables
+DECLARE @f FLOAT = 123.456;
+DECLARE @r REAL = 78.90;
+EXEC bbf_rpe_server.master.dbo.sp_TestFloat @p_float=@f, @p_real=@r;
+GO
+~~START~~
+float#!#real
+123.456#!#78.9
+~~END~~
+
+
+-- Test 8: DECIMAL and NUMERIC with literals
+-- Expected: Should preserve precision and scale
+EXEC bbf_rpe_server.master.dbo.sp_TestDecimal @p_decimal=1234.5678, @p_numeric=98.76;
+GO
+~~START~~
+numeric#!#numeric
+1234.5678#!#98.76
+~~END~~
+
+
+-- Test 9: DECIMAL with typed variable (regression test for OID issue)
+-- Expected: Should correctly handle DECIMAL typed variables
+DECLARE @d DECIMAL(18,4) = 9999.9999;
+DECLARE @n NUMERIC(10,2) = 12345.67;
+EXEC bbf_rpe_server.master.dbo.sp_TestDecimal @p_decimal=@d, @p_numeric=@n;
+GO
+~~START~~
+numeric#!#numeric
+9999.9999#!#12345.67
+~~END~~
+
+
+-- Test 10: MONEY and SMALLMONEY types
+-- Expected: Should handle money types correctly
+EXEC bbf_rpe_server.master.dbo.sp_TestMoney @p_money=1234.5678, @p_smallmoney=99.99;
+GO
+~~START~~
+money#!#smallmoney
+1234.5678#!#99.9900
+~~END~~
+
+
+-- Test 11: MONEY with variables
+-- Expected: Should handle MONEY typed variables
+DECLARE @m MONEY = 99999.9999;
+DECLARE @sm SMALLMONEY = 214748.3647;
+EXEC bbf_rpe_server.master.dbo.sp_TestMoney @p_money=@m, @p_smallmoney=@sm;
+GO
+~~START~~
+money#!#smallmoney
+99999.9999#!#214748.3647
+~~END~~
+
+
+
+-- ========================================
+-- SECTION 3: STRING DATA TYPES
+-- ========================================
+-- Test 12: VARCHAR parameter
+-- Expected: Should pass string correctly
+EXEC bbf_rpe_server.master.dbo.sp_TestVarchar @p='Hello World';
+GO
+~~START~~
+varchar#!#int
+Hello World#!#11
+~~END~~
+
+
+-- Test 13: VARCHAR with variable
+-- Expected: Should handle VARCHAR variable
+DECLARE @str VARCHAR(100) = 'Testing VARCHAR parameter';
+EXEC bbf_rpe_server.master.dbo.sp_TestVarchar @p=@str;
+GO
+~~START~~
+varchar#!#int
+Testing VARCHAR parameter#!#25
+~~END~~
+
+
+-- Test 14: NVARCHAR parameter
+-- Expected: Should handle Unicode string
+EXEC bbf_rpe_server.master.dbo.sp_TestNvarchar @p=N'Unicode Test String';
+GO
+~~START~~
+varchar#!#int
+Unicode Test String#!#19
+~~END~~
+
+
+-- Test 15: NVARCHAR with special characters
+-- Expected: Should preserve special characters
+DECLARE @nstr NVARCHAR(100) = N'Special: !@#$%^&*()_+-={}[]|:;<>?,./';
+EXEC bbf_rpe_server.master.dbo.sp_TestNvarchar @p=@nstr;
+GO
+~~START~~
+varchar#!#int
+Special: !@#$%^&*()_+-={}[]|:;<>?,./#!#36
```

Comment:
These are not special characters for nvarchar, these are ascii characters which will work for varchar. Please tests for nvarchar specific chars too like emojis and kanji characters